slateratom/lib/confinement.F90

```diff
@@ -0,0 +1,514 @@
+#:include 'common.fypp'
+
+!> Module that builds up various supervectors.
```

Comment:
Not sure if that covers what this is doing...
```suggestion
!> Module that builds up various confining potential supervectors.
```

Reply:
More accurate, looks good :+1:.